apps/app/src/app/components/charts/bar-chart/bar-chart.component.html

```diff
@@ -1,3 +1,8 @@
 <div class="bar-chart"
      #container
-     [id]="id"></div>
\ No newline at end of file
+     [id]="id">
+  <div class="tooltip"
+       [style.opacity]="showTooltip ? '1' : '0'"
+       [style.bottom.px]="tooltipBottom"
+       [style.left.px]="tooltipLeft">{{ (tooltipContent || '').toLocaleString() }}</div>
```

Comment:
same here: tooltip content is a number, so better to use DecimalPipe from angular